components/script/dom/audioparam.rs

```diff
@@ -190,6 +190,24 @@ impl AudioParamMethods for AudioParam {
         DomRoot::from_ref(self)
     }
 
+    // https://webaudio.github.io/web-audio-api/#dom-audioparam-setvaluecurveattime
+    fn SetValueCurveAtTime(
+        &self,
+        values: Vec<Finite<f32>>,
+        start_time: Finite<f64>,
+        end_time: Finite<f64>,
+    ) -> DomRoot<AudioParam> {
+        self.message_node(AudioNodeMessage::SetParam(
```

Comment:
This needs to do a bunch of input validation, see the "description" column of the arguments table on https://webaudio.github.io/web-audio-api/#dom-audioparam-setvaluecurveattime

Reply:
For this, I based my code on the other processing algorithms that are already present in AudioParam.rs because they had the same kind of validation description. Can you give me some pointers or some example that I can look into so that I can fix the same? 

Thanks.

Reply:
You can see examples of such checks on https://github.com/ferjm/servo/blob/webaudio/components/script/dom/audionode.rs#L100

(Looks like the other AudioParam functions are missing validation too, I should add that)

Reply:
You can see examples of such checks on https://github.com/ferjm/servo/blob/webaudio/components/script/dom/audionode.rs#L100

(Looks like the other AudioParam functions are missing validation too, I should add that)

Reply:
do the same thing done in https://github.com/servo/servo/pull/23270

Reply:
I have added the validations and pushed the code. 

Thanks.

Reply:
I think I have corrected the error, can you take a look now?